src/components/data-exploration/heatmap/HeatmapLegendVisibilitySettings.jsx

```diff
@@ -1,48 +0,0 @@
-import React, { useState } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import { Radio } from 'antd';
-import PropTypes from 'prop-types';
-
-import { updatePlotConfig } from '../../../redux/actions/componentConfig';
-
-const HeatmapLegendVisibilitySettings = (props) => {
```

Comment:
Why do we no longer need this file?

Reply:
Because the new type of expression values we show is calculated in a way that is gene-specific, so we can't show a bar saying "this intensity of color means a gene's expression in a cell is 3" universally because there is a different bar of intensity for each gene. This was discussed and agreed on with vicky and alex already.